docs/user/metric-parameters.md

```diff
@@ -27,7 +27,7 @@
 
 - `lifetime`: Defines the lifetime of the metric. Different lifetimes affect when the metrics value is reset.
   - `ping` (default): The metric is reset each time it is sent in a ping.
-  - `application`: The metric is related to an application run, and is reset when the application restarts.
+  - `application`: The metric is related to an application run, and is reset when the application restarts. Care should be taken when using this lifetime: if the metric is sent with the [`metrics` ping](pings/metrics.md), set its value as early as possible to minimise the possibility of it being missing from the ping.
```

Comment:
nit: `minimise` would probably be correct for British English, but I believe the more common spelling would be `minimize`

```suggestion
  - `application`: The metric is related to an application run, and is reset when the application restarts. Care should be taken when using this lifetime: if the metric is sent with the [`metrics` ping](pings/metrics.md), set its value as early as possible to minimize the possibility of it being missing from the ping.
```

Reply:
In absence of a style guide, I think you can use whichever form you'd like. Doesn't even have to be English : D

(( fwiw, Canadian English goes with `ize` in most cases as well, so I probably have a bias here. ))

Reply:
I'm also biased, which is why I relegated this to the level of "nit" :)